juniper_official/Linecard/aft-server-entry-stats.rule

```diff
@@ -0,0 +1,41 @@
+healthbot {
+    topic external {
+        rule aft_server_entry_stats {
+            synopsis "Aft Server Sandbox Entry Stats";
+            description "Aft Server Sandbox Entry Stats";
+            sensor aft_sb_entry_stats {
+                iAgent {
+                    file aft-sandbox-stats.yml;
```

Comment:
Sensor Synopsis and description is mandatory, please include

Reply:
Done